mappings/net/minecraft/util/registry/RegistryCodecs.mapping

```diff
@@ -0,0 +1,78 @@
+CLASS net/minecraft/class_6895 net/minecraft/util/registry/RegistryCodecs
+	METHOD method_40338 (Lnet/minecraft/class_2378;)Ljava/util/Map;
+		ARG 0 registry
+	METHOD method_40339 (Lnet/minecraft/class_2385;Lcom/mojang/serialization/Lifecycle;Lnet/minecraft/class_5321;Ljava/lang/Object;)V
+		ARG 2 key
+		ARG 3 value
+	METHOD method_40340 createRegistryEntryListCodec (Lnet/minecraft/class_5321;)Lcom/mojang/serialization/Codec;
```

Comment:
Very minor thought: shorter-style names might be nice here- e.g. `RegistryCodecs.entryList(...)` instead of `RegistryCodecs.createRegistryEntryListCodec(...)` where there is a bit of redundancy.
I imagine these are used quite often in record codec builders, so it might help to reduce noise? 🙂